tsconfig.json

```diff
@@ -37,8 +37,7 @@
     "ads",
     "extensions",
     "test",
-    "testing",
-    "third_party"
+    "testing"
```

Comment:
Looks like this needs to run `amp prettier --local_changes --fix` to reformat for the check tests: https://app.circleci.com/pipelines/github/ampproject/amphtml/15834/workflows/a0f16f41-ecc9-4c24-a4c9-e09c04244f97/jobs/278590

Reply:
fixed